packages/cli/generators/datasource/index.js

```diff
@@ -287,6 +288,30 @@ module.exports = class DataSourceGenerator extends ArtifactGenerator {
     // Copy Templates
     this.fs.writeJSON(jsonPath, ds);
     this.copyTemplatedFiles(classTemplatePath, tsPath, this.artifactInfo);
+
+    if (ds.connector === 'memory') {
+      if (!this.artifactInfo.settings.file) {
```

Comment:
As I commented above, it's perfectly fine and often desirable to leave the `file` option not set.